pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
Another reason it may not make sense to merge them is that the messages added here are specifically designed to be a kubectl enhancement - see #3937. They wouldn't necessarily make sense outside kubectl context.